api/test/wfl/integration/modules/covid_test.clj

```diff
@@ -95,8 +109,70 @@
       (is (contains? (set labels) (str "pipeline:" covid/pipeline)))
       (is (vector? watchers)))))
 
+(defn ^:private make-covid-workload-request []
+  (-> (workloads/covid-workload-request {} {} {})
+      (assoc :creator @workloads/email)))
+
+;(deftest test-create-covid-workload
+;  (testing "That COVID workload-request can pass verification"
+;    (let [workload (workloads/create-workload! (make-covid-workload-request))]
+;      (is workload))))
+
+(deftest test-create-covid-workload-with-misnamed-source
+  (testing "That a COVID workload-request with a misnamed source cannot pass verification"
```

Comment:
Nit: all of your broken-up tests are named descriptively enough that I don't think you need the `(testing …)` wrappers.

Reply:
I removed this line as suggested.

Reply:
Sorry, has that change been pushed to the branch yet? I'm not seeing it. (You might have done this as well, but I'll note that this was a suggestion for all of your newly added tests.)

Reply:
Sorry. Hadn't pushed it yet. Now it is pushed